### ISSUE TITLE
Bucket with id '-5' was not found

### ISSUE THREAD

**Issue by ExtReMLapin** [ArcadeData/arcadedb#1477]
Hello,

very likely related to #947 

### ArcadeDB Version:

Latest release 24.1.1

### OS and JDK Version:

Tried on both windows and ubuntu

` Running on Windows 10 10.0 - Java HotSpot(TM) 64-Bit Server VM 21.0.1`

### Expected behavior

SUMMARY entities to be removed

### Actual behavior
```
févr. 23 11:42:23 cfia01 server.sh[1386844]: com.arcadedb.exception.CommandParsingException: Error on executing Cypher query
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.cypher.query.CypherQueryEngine.command(CypherQueryEngine.java:80)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.database.LocalDatabase.command(LocalDatabase.java:1348)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.server.ServerDatabase.command(ServerDatabase.java:472)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.server.http.handler.PostCommandHandler.executeCommand(PostCommandHandler.java:131)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.server.http.handler.PostCommandHandler.execute(PostCommandHandler.java:110)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.server.http.handler.DatabaseAbstractHandler.execute(DatabaseAbstractHandler.java:100)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.server.http.handler.AbstractServerHttpHandler.handleRequest(AbstractServerHttpHandler.java:127)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at io.undertow.server.Connectors.executeRootHandler(Connectors.java:393)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:859)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.xnio.XnioWorker$WorkerThreadFactory$1$1.run(XnioWorker.java:1282)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at java.base/java.lang.Thread.run(Thread.java:829)
févr. 23 11:42:23 cfia01 server.sh[1386844]: Caused by: com.arcadedb.exception.SchemaException: Bucket with id '-5' was not found
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.schema.LocalSchema.getBucketById(LocalSchema.java:289)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.database.LocalDatabase.lambda$lookupByRID$14(LocalDatabase.java:687)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.database.LocalDatabase.executeInReadLock(LocalDatabase.java:1415)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.database.LocalDatabase.lookupByRID(LocalDatabase.java:668)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.database.RID.asEdge(RID.java:154)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.graph.EdgeIterator.next(EdgeIterator.java:84)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.graph.EdgeIterator.next(EdgeIterator.java:30)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.graph.GraphEngine.deleteVertex(GraphEngine.java:391)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.database.LocalDatabase.deleteRecordNoLock(LocalDatabase.java:1005)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.database.LocalDatabase.lambda$deleteRecord$20(LocalDatabase.java:973)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.database.LocalDatabase.executeInReadLock(LocalDatabase.java:1415)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.database.LocalDatabase.deleteRecord(LocalDatabase.java:972)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.gremlin.ArcadeGraph.deleteElement(ArcadeGraph.java:449)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.gremlin.ArcadeElement.remove(ArcadeElement.java:89)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.filter.DropStep.filter(DropStep.java:67)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.filter.FilterStep.processNextStart(FilterStep.java:41)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.hasNext(AbstractStep.java:155)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.util.DefaultTraversal.hasNext(DefaultTraversal.java:192)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.map.CoalesceStep.flatMap(CoalesceStep.java:58)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.map.FlatMapStep.processNextStart(FlatMapStep.java:49)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.hasNext(AbstractStep.java:155)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.util.DefaultTraversal.hasNext(DefaultTraversal.java:192)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.sideEffect.TraversalSideEffectStep.sideEffect(TraversalSideEffectStep.java:48)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.sideEffect.SideEffectStep.processNextStart(SideEffectStep.java:39)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.hasNext(AbstractStep.java:155)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.util.ExpandableStepIterator.hasNext(ExpandableStepIterator.java:47)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.map.NoOpBarrierStep.processAllStarts(NoOpBarrierStep.java:67)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.map.NoOpBarrierStep.processNextStart(NoOpBarrierStep.java:56)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.hasNext(AbstractStep.java:155)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.util.ExpandableStepIterator.next(ExpandableStepIterator.java:55)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.filter.FilterStep.processNextStart(FilterStep.java:40)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.step.util.AbstractStep.hasNext(AbstractStep.java:155)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at org.apache.tinkerpop.gremlin.process.traversal.util.DefaultTraversal.hasNext(DefaultTraversal.java:192)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.gremlin.ArcadeGremlin$2.hasNext(ArcadeGremlin.java:112)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.query.sql.executor.IteratorResultSet.hasNext(IteratorResultSet.java:37)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.cypher.ArcadeCypher.execute(ArcadeCypher.java:75)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         at com.arcadedb.cypher.query.CypherQueryEngine.command(CypherQueryEngine.java:77)
févr. 23 11:42:23 cfia01 server.sh[1386844]:         ... 13 more

```

### Steps to reproduce

[Test.zip](https://github.com/ArcadeData/arcadedb/files/14384820/Test.zip)

download db linked in message, and run `match(n:SUMMARY) detach delete n` cypher command on DB

It's my coworker that has this issue, I still don't think she does something really exotic

**Comment by ExtReMLapin** [comment]
Could be also related to #1476 as she tried the `match (n) detach delete n` that returned the error from the issue  #1476 so she tried to delete "classes" one by one which leaded to this error.



Long story short, you can both `match (n) detach delete n` on this database and `match(n:SUMMARY) detach delete n` you'll have two different errors

**Comment by ExtReMLapin** [comment]
Please note we're working on a RAG as you may have guessed by the database. She's a junior so she "spams" requests and as the other issue guy talks about a race condition, i'm wondering if it could be that aswell here.

**Comment by ExtReMLapin** [comment]
Does it need to be reconfirmed, with #1476 being fixed ?

**Comment by lvca** [comment]
This is likely fixed by #1476. Closing this, if you can still reproduce it, please comment here to reopen it. Thanks @ExtReMLapin.
